apps/simple-trading-app/src/app/components/header/header.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { ThemeSwitcher } from '@vegaprotocol/ui-toolkit';
+import Video from './video';
+import Logo from './logo';
+import Comet from './comet';
+import { VegaWalletConnectButton } from '../vega-wallet-connect-button';
+
+type WalletParams = {
+  connect: boolean;
+  manage: boolean;
+};
+
+interface Props {
+  setVegaWallet: (func: (wParams: WalletParams) => WalletParams) => void;
+  toggleTheme: () => void;
+}
+const Header = ({ setVegaWallet, toggleTheme }: Props) => {
+  return (
+    <div
+      className="flex items-stretch p-16 bg-black text-white-60"
+      data-testid="header"
+    >
+      <div className="absolute top-0 right-[200px] w-[500px] h-[100px] z-0 hidden md:block">
+        <Video />
+        <div
+          id="swarm"
+          className="-top-[1px] -left-[1px] absolute w-[502px] h-[102px] bg-black"
```

Comment:
IMHO it will be better to use `-bottom-[1px] -right-[1px]` instead of `w-[502px] h-[102px]`

Reply:
Actually it was not quite by last version of design, so I had to remove it :-(